.github/workflows/release_validate.yml

```diff
@@ -20,7 +20,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: release # Use the 'release' branch even if triggered by cron.
+          ref: release
+      # Run on the latest tag. The release branch is ahead of release tag over the weekend.
+      - run: |
+          TAG=$(gh release list --limit 1 | cut -f1)
```

Comment:
Ideally, we'd pass that as a `ref` above, but I couldn't get it to work, and the docs are strategically ambiguous regrarding whether that's possible at all.